lucene/core/src/test/org/apache/lucene/index/TestIndexWriterReader.java

```diff
@@ -169,7 +176,7 @@ public void testUpdateDocument() throws Exception {
     // writer.close wrote a new commit
     assertFalse(r2.isCurrent());
 
-    DirectoryReader r3 = DirectoryReader.open(dir1);
+    DirectoryReader r3 = open(dir1);
```

Comment:
Can you keep the explicit version ?

Reply:
Addressed in https://github.com/apache/lucene/pull/32/commits/f3fbba10d5ff391bc66bbd04032a873c91680493